platforms/cpuinfo.go

```diff
@@ -97,10 +125,21 @@ func getCPUVariant() string {
 
 	variant, err := getCPUInfo("Cpu architecture")
 	if err != nil {
-		log.L.WithError(err).Error("failure getting variant")
-		return ""
+		//If we can't find variant from /proc/cpuinfo, let's try uname -m.
+		//This issue was seen when server and client running in different architectures
```

Comment:
Let's explain why in addition to the situation.

Reply:
Comment updated with explanation.